Libraries/LibChess/Chess.h

```diff
@@ -143,13 +154,17 @@ class Board {
         NotFinished,
     };
 
+    String result_to_string(Result) const;
+    String result_to_points(Result) const;
```

Comment:
It's weird that these functions take a `Result`, but are not `static` since they use `m_turn`.

This means that to get the string representation of a board `b`, you have to call `result_to_string` function as:
```cpp
auto stringified_result = b.result_to_string(b.game_result());
```

Why don't you add to these functions a `Colour` (turn) parameter and make them `static` (or external helper functions)?

You could also just remove the `Result` argument and make these functions call `game_result()` internally.